src/site/topic22.rst

```diff
@@ -30,6 +30,14 @@ Level-order
 Iterative Pre/In/Post-Order
 ---------------------------
 
+* With the iterative level-order traversal, we used a *queue*
+* What do you think would happen if we switched it out for a stack?
+
+* With the recursive pre-/in-/post-order traversals, we used the call stack
+    * We didn't need a software stack, but we could have used one
```

Comment:
[Software stack](https://en.wikipedia.org/wiki/Software_stack) means something totally different nowadays. Definitely prefer to specify the _stack data structure_ (or abstract data type, just not software).